### PR TITLE
Example usd-materialx files

### PR DESCRIPTION
**Pull request by ashwinbhat**
This PR introduces a few simple usd files that use MaterialX 
Example 1: Basic 
Use a built-in USD Sphere with usd_preview_surface_plastic.mtlx
Example 2: Basic Textured with variants
Use a teapot model referring 2 MaterialX files usd_preview_surface_plastic.mtlx and standard_surface_brass_tiled.mtlx 

Also see [readme](https://github.com/ashwinbhat/assets/blob/main/test_assets/MaterialXTest/readme.md) for details.

### CODE REVIEW

**Comment by AlexSchwank** [comment]
Double checking, where's the teapot from? Do we have the rights to have it in here? If so, it would be useful to have it in the `_common` folder so that other tests can use it.

**Comment by ashwinbhat** [comment]
I converted the teapot from teapot.obj from MaterialX repo [here](https://github.com/AcademySoftwareFoundation/MaterialX/tree/main/resources/Geometry)
I'll move it to common, if we can reuse.

**Comment by AlexSchwank** [comment]
Do you have screenshots of all files you can add to the readme?

File: test_assets/MaterialXTest/readme.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+To enable materialx, build USD with `--materialx` option
+
+```python build_scripts\build_usd.py  --materialx ```
+
+### Sample USD files
+For details on how to use MaterialX in USD see ASWF Open Source 2021 [talk](https://materialx.org/assets/ASWF_OSD2021_MaterialX_slides_final.pdf).
+
+The MaterialX files and teapot model referenced in these samples are from ASWF MaterialX [repository](https://github.com/AcademySoftwareFoundation/MaterialX/tree/main/resources/Materials/Examples). 
+
```
Comment (meshula) [inline test_assets/MaterialXTest/readme.md:13]:
```suggestion
The MaterialX files and teapot model are TM & (c) 2017 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
All rights reserved, and covered by the Apache 2.0 license.
```

**Review by meshula — CHANGES_REQUESTED** [review]

File: test_assets/MaterialXTest/readme.md (hunk 1 above)
Comment (AlexSchwank) [reply test_assets/MaterialXTest/readme.md:13]: The teapot note should go into `_common/README.md` ?

**Review by AlexSchwank — APPROVED** [review]

**Pull request merged**